apps/server/src/shared/domain/entity/task-card.entity.ts

```diff
@@ -39,10 +40,22 @@ export class TaskCard extends BaseEntityWithTimestamps implements ICard, ITaskCa
 	@Property()
 	draggable = true;
 
+	@Property({ nullable: true })
+	completionDate?: Date;
+
 	public getCardElements() {
 		return this.cardElements.getItems();
 	}
 
+	public pastCompletionDate(): boolean {
```

Comment:
I would rename it to somethiing indicating boolean return like isPastCompletionDate()